config/config.go

```diff
@@ -100,6 +100,9 @@ type Config struct {
 	DelayCleanTableLock uint64      `toml:"delay-clean-table-lock" json:"delay-clean-table-lock"`
 	SplitRegionMaxNum   uint64      `toml:"split-region-max-num" json:"split-region-max-num"`
 	StmtSummary         StmtSummary `toml:"stmt-summary" json:"stmt-summary"`
+	// RepairMode indicates that the TiDB is in the repair mode for table meta.
+	RepairMode      bool     `toml:"repair-mode" json:"repair-mode"`
```

Comment:
I think it’s better to bootstrap repair mode with CLI arguments instead of configuration.

Reply:
Addressed.

Reply:
Do we need this configuration if we have CLI parameters? @djshow832 @bb7133

Reply:
It's OK here. We can offer users with another choice.